nautobot/dcim/models/device_components.py

```diff
@@ -568,6 +584,8 @@ class Interface(CableTermination, PathEndpoint, ComponentModel, BaseInterface):
         "device",
         "name",
         "label",
+        "parent_interface",
+        "bridge",
```

Comment:
I'm not sure I can think of any backward compatibility issue

Reply:
Yes, that was precisely my concern - if someone's naively expecting the CSV columns to follow a constant order (rather than actually checking the column headers) this seemed like it might pose a problem.